_schemaV2.graphql

```diff
@@ -1509,7 +1509,7 @@ type ArticlesRailHomeViewSection implements GenericHomeViewSection & Node {
     before: String
     first: Int
     last: Int
-  ): ArticleConnection!
+  ): ArticleConnection
```

Comment:
I made ArticleConnection and FairConnection nullable to align with other connections